<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jan 'Kovis' Struhar</author>
 <class>ServiceDetailDlg</class>
 <widget class="QDialog" name="ServiceDetailDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>488</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Holy service detail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="4" column="3" colspan="6">
      <widget class="QLabel" name="clientDescriptionLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;no client assigned&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="clientLabel">
       <property name="text">
        <string>Client:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="priestCombo">
       <property name="toolTip">
        <string>Priest that shall celebrate the service</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="dateDisplay">
       <property name="text">
        <string>Service date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Priest:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="changeDateButton">
       <property name="toolTip">
        <string>Shift this service to different date</string>
       </property>
       <property name="text">
        <string>Change date</string>
       </property>
       <property name="icon">
        <iconset resource="HolyServices.qrc">
         <normaloff>:/images/changeDate.png</normaloff>:/images/changeDate.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="churchCombo">
       <property name="toolTip">
        <string>Church where the holy service is to take place in</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="churchLabel">
       <property name="text">
        <string>Church:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="clientCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clearClientAssignmentButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Clear client assignment</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="HolyServices.qrc">
           <normaloff>:/images/clear_left.png</normaloff>:/images/clear_left.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newClientBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add new client record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="HolyServices.qrc">
           <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="4" colspan="6">
      <widget class="QLabel" name="churchDescriptionLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;no church assigned&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="6">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="fromTimeLabel">
         <property name="text">
          <string>From</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="fromTimeEdit">
         <property name="displayFormat">
          <string>H:mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string>To</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="endTimeEdit">
         <property name="displayFormat">
          <string>H:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>21</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="intentionLabel">
       <property name="text">
        <string>Intention:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FocusReportingEdit" name="intentionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Received:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="moneyBox">
       <property name="toolTip">
        <string>Mass contribution</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="fixedToDateButon">
         <property name="toolTip">
          <string>Client insists on given date</string>
         </property>
         <property name="text">
          <string>Fixed to date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="movableButton">
         <property name="toolTip">
          <string>Under circumstances the client agrees to move holy service to different date</string>
         </property>
         <property name="text">
          <string>Movable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="historicalInfoButton">
       <property name="toolTip">
        <string>Shows history records about currently edited holy service</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="HolyServices.qrc">
         <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FocusReportingEdit</class>
   <extends>QTextEdit</extends>
   <header>FocusReportingEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>churchCombo</tabstop>
  <tabstop>priestCombo</tabstop>
  <tabstop>changeDateButton</tabstop>
  <tabstop>clientCombo</tabstop>
  <tabstop>intentionEdit</tabstop>
  <tabstop>moneyBox</tabstop>
  <tabstop>fixedToDateButon</tabstop>
  <tabstop>movableButton</tabstop>
  <tabstop>historicalInfoButton</tabstop>
 </tabstops>
 <resources>
  <include location="HolyServices.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServiceDetailDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
